src/assets/scss/components/_search.scss

```diff
@@ -9,4 +9,36 @@
       margin: .25rem .5rem .25rem 0;
     }
   }
+
+  @media only screen and (max-width: 767px) {
+    &__topic {
+      position: relative;
+    }
+
+    &__magnifyingGlass {
```

Comment:
@JacOng17 Is camel casing used anywhere else in the code? If not we should follow the convention ie magnifying-glass (if thats how it is)

Reply:
Not too sure, will have a look if not will correct it :+1: